wpilib/wpilib/relay.py

```diff
@@ -266,30 +265,16 @@ def setDirection(self, direction):
         if self.direction == direction:
             return
         
-        if direction not in [self.Direction.kBoth,
-                             self.Direction.kForward,
-                             self.Direction.kReverse]:
+        if direction not in self.Direction :
```

Comment:
Firstly, there is an extra space before the `:`.

Secondly, this breaks users who are currently passing integer constants rather than referencing `Relay.Direction.foo`. I don't think we want to break that until after a year of explicitly deprecating that usage.